src/Benchmarks/Startup.cs

```diff
@@ -58,7 +60,7 @@ public void ConfigureServices(IServiceCollection services)
 
             if (Scenarios.Any("Ef"))
             {
-                services.AddScoped<EfDb>();
+                services.AddTransient<EfDb>();
```

Comment:
Why was EF changed from scoped (once per http request) to transient (once per service request)?